tests/ci/cdk/app.py

```diff
@@ -50,5 +51,7 @@
 # TODO: re-enable 'aws-lc-ci-bm-framework' when it's ready.
 # bm_framework_build_spec_file = "./cdk/codebuild/bm_framework_omnibus.yaml"
 # BmFrameworkStack(app, "aws-lc-ci-bm-framework", LINUX_X86_ECR_REPO, bm_framework_build_spec_file, env=env)
+android_build_spec_file = "cdk/codebuild/github_ci_android_omnibus.yaml"
+AwsLcAndroidCIStack(app, "aws-lc-devicefarm-android", LINUX_X86_ECR_REPO, android_build_spec_file, env=env)
```

Comment:
Can this dimension get merged to x86 omnibus? GitHub has 20 webhook limits. New webhook/batch build stack should be avoided to leave spaces for future. Other stacks are separated due to various reasons(e.g. Fuzz one require EFS, ARM one requires ARM instance).

Reply:
> I think we can merge it in with x86 omnibus.

https://github.com/awslabs/aws-lc/pull/448#discussion_r841955279